panelStyle.js

```diff
@@ -273,7 +273,7 @@ var PanelStyle = class {
     }
     
     _overrideStyle(actor, styleLine, operationIdx) {
-        if (actor._dtp_original_inline_style === undefined) {
+        if (actor._dtp_original_inline_style === undefined && actor.get_style !== undefined) {
```

Comment:
Maybe we could validate that actor is an St.Widget in the compareFn that checks the parent? I'm guessing that in PaperWM the actor is a ClutterActor?

Reply:
wouldn't that cause duplicated code? There's a lot of checks that lead to `_overrideStyle` and I can't remember which one(s) came up